init.sh

```diff
@@ -1,11 +1,13 @@
 # Detect os
-TOOLSPATH=/cvmfs/fcc.cern.ch/sw/0.8.3/tools/
-OS=`python $TOOLSPATH/hsf_get_platform.py --get os`
+# Disabled as currently there is only one platform supported
+#TOOLSPATH=/cvmfs/fcc.cern.ch/sw/0.8.3/tools/
+#OS=`python $TOOLSPATH/hsf_get_platform.py --get os`
 
-# source FCC externals
-source /cvmfs/fcc.cern.ch/sw/views/releases/externals/96c_LS.0.0/x86_64-$OS-gcc8-opt/setup.sh
+# source LCG releases
+source /cvmfs/sft.cern.ch/lcg/views/LCG_96c_LS/x86_64-centos7-gcc8-opt/setup.sh
```

Comment:
I think it may be a good idea to keep this script working also for the usecase of setting up the environment for compiling dd4hep on, say, lxplus; $LCG_RELEASE and $COMPILER_VERSION are specific to Travis. 
But I changed the script to set those env vars and use them in the path.

Reply:
You can use a default value for the variables
```
${VARIABLE:-DEFAULT}
${LCG_RELEASE:-LCG_96c_LS}
${OS:-centos7}
${COMPILER_VERSION:-gcc8}
```
Or `:=` instead of `:-` if the parameters are used many times, which also assigns the default to the variable, so at the next use the default is used without setting defaults everywhere.

And there is the logic here:
https://github.com/AIDASoft/DD4hep/blob/master/.dd4hep-ci.d/init_x86_64.sh
to switch between slc6 and centos7 
Also some logic to be able to use dev installations from sft-nightlies

Reply:
Great suggestion! 

>to switch between slc6 and centos7
Will it be necessary to build on slc6?

Reply:
I am still using SLC6

Reply:
Ok. I'd propose to extend the CI to multiple platforms in another PR?

Reply:
Yes that is fine.